graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -504,15 +512,16 @@ def _check_field_reference_is_valid(type_name_to_definition, type_name_to_schema
 
     Args:
         type_name_to_definition: Dict[str, (Interface/Object)TypeDefinition], mapping
-                                 name of types to their definitions
-        type_name_to_schema_id: Dict[str, str], mapping type name to the id of the schema that the
-                                type is from
+                                 name of Interfae and Object types to their definitions
```

Comment:
typo: `Interfae`, possibly a bad find-and-replace?

Reply:
A bad copy paste of Args description, sorry about that